src/main/resources/application.properties

```diff
@@ -3,7 +3,7 @@ spring.jpa.hibernate.dll-auto=update
 spring.jpa.properties.hibernate.dialect=org.hibernate.dialect.MySQL8Dialect
 spring.datasource.url=jdbc:mysql://localhost:3306/hospital_appointment_management-db?createDatabaseIfNotExist=true&autoReconnect=true&useSSL=false&allowPublicKeyRetrieval=true
 spring.datasource.username=root
-spring.datasource.password=password
+spring.datasource.password=
```

Comment:
Please do not not change the password for the db, you can google how to change your databases password on wamp and for that specific database as well to the password that is already stated there